packages/quantic/cypress/integration/tab/tab.cypress.ts

```diff
@@ -98,4 +100,31 @@ describe('quantic-tab', () => {
       Expect.activeTabContains(tabs.knowledge.label);
     });
   });
+
+  describe('when using accessibility', () => {
```

Comment:
The cypress tests for Quantic are written to avoid reloading the page as much as possible as it slows down test runs significantly. 
To do so we try to use as few `it()` blocks as possible and instead separate tests with `scope()` blocks within `it()`s.
If you look at other tests like the facet tests it'll be clearer.
So unless you're loading the tabs with different options or from a different url hash, it might be possible to package several tests and assertions like that without reloading.

Reply:
Great improvement!! Thanks!